doc/rbd/rbd-mirroring.rst

```diff
@@ -0,0 +1,233 @@
+===============
+ RBD Mirroring
+===============
+
+.. index:: Ceph Block Device; mirroring
+
+RBD images can be asynchronously mirrored between two Ceph clusters.  This
+capability uses the RBD journaling image feature to ensure crash-consistent
+replication between clusters.  Mirroring is configured on a per-pool basis
+within peer clusters and can be configured to automatically mirror all pool
+images within a pool or only a specific subset of images.  Mirroring is
+configured using the ``rbd`` command.  The ``rbd-mirror`` daemon is responsible
+for pulling image updates from the remote, peer cluster and applying them to
+the image within the local cluster.
+
+.. important:: To use use RBD mirroring, you must have two Ceph clusters and
+   the ability for the ``rbd-mirror`` daemon to connect to both clusters
+   simultaneously.
```

Comment:
This makes it sound like there is only a single `rbd-mirror` daemon for both clusters. After reading the document I'm confused about where this daemon actually runs. I assume a single instance of the daemon runs on each of the peer clusters, but I'm not sure.